rfcs/47-better-error-handing.md

```diff
@@ -0,0 +1,251 @@
+---
+author: xxchan <xxchan22f@gmail.com>
+status: draft
+updated_at: 2020-05-03
+---
+
+# AOS-47: Better Error Handling
```

Comment:
We need a better proposal name here, a title that describes what has been changed in this proposal.

Reply:
What about: Additional error specification

Reply:
LGTM